lambdarest/__init__.py

```diff
@@ -106,18 +107,24 @@ def inner_lambda_handler(event, context=None):
         path = event["path"].lower()
         method_name = event["httpMethod"].lower()
         func = None
+        kwargs = {}
         error_tuple = ("Internal server error", 500)
         logging_message = "[%s][{status_code}]: {message}" % method_name
         try:
-            func = http_methods.get(path, http_methods.get("*", {}))[method_name]
+            mapping = url_maps.bind('example.com', '')
```

Comment:
what does `example.com` have to do here?

Reply:
I figured we needed an server_name to bind it, but it's actually not needed at all, removed it.